README.md

```diff
@@ -1,14 +1,57 @@
 # Getting Started with the .NET MAUI Text Input Layout
-A quick-start project that helps you to integrate a Text input layout control in the .NET MAUI application. This project contains simple code customizations like how to add floating labels, password toggle icons to show or hide passwords, and assistive labels like error messages and help text on top of input controls.
-
-Refer to the following documentation for the Syncfusion .NET MAUI Text Input Layout control: 
-https://help.syncfusion.com/maui/textinputlayout/getting-started
-
-Check out this example of the Syncfusion .NET MAUI Text Input control on GitHub: 
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/TextInputLayout
+The [.NET MAUI Text Input Layout](https://www.syncfusion.com/maui-controls/maui-textinputlayout?utm_source=github&utm_medium=listing&utm_campaign=maui-text-input-layout-github-samples) or Label Entry is a container control that allows users to add floating labels, password toggle icons to show or hide passwords, leading and trailing icons, and assistive labels such as error messages and help text on top of input controls.A quick-start project that helps you to integrate a Text input layout control in the .NET MAUI application. This project contains simple code customizations like how to add floating labels, password toggle icons to show or hide passwords, and assistive labels like error messages and help text on top of input controls.

```

Comment:
Give a space here..
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-text-input-layout/assets/95272441/0db14f9d-b437-4197-8557-d93471f108b1)

Reply:
Done